manticore/platforms/linux.py

```diff
@@ -312,6 +315,11 @@ def __init__(self, program, argv=None, envp=None, disasm='capstone', **kwargs):
         self.programs = program
         self.disasm = disasm
 
+        # dict of [int -> (int, int)] where tuple is (soft, hard) limits
```

Comment:
if we wanted to get extra fancy, we could use a NamedTuple here and omit the comment describing the tuple. now or later, either is fine though.